kroxylicious-kms-provider-kroxylicious-inmemory-test-support/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
+        <version>0.4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>kroxylicious-kms-provider-kroxylicious-inmemory-test-support</artifactId>
+    <description>Test support code for modules testing the InMemory KMS</description>
+
+    <dependencies>
+        <!-- project dependencies - runtime and compile -->
+        <dependency>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious-kms-provider-kroxylicious-inmemory</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious-kms-test-support</artifactId>
+        </dependency>
```

Comment:
This module should depend on `junit-jupiter-api` and `assertj-core` as it directly uses them (Yes, I know they are available transitively from `kroxylicious-kms-test-support`).

Alternatively we could make `kroxylicious-kms-test-support` a parent module but I'm not sure about what that implies for third party implementors and our general maven usage pattern.

Reply:
We should do the direct dependency as part of this PR. anything wider ranging should happen separately.

Reply:
DOne.